server/resource_server/controller/controller_test.go

```diff
@@ -673,6 +853,196 @@ func TestLoginUserHandler_Success(t *testing.T) {
 	assert.Equal(t, "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VybmFtZSI6ImhtayIsInVzZXJfaWQiOjIsImlzcyI6Ikdsb2JlQW5kQ2l0aXplbiIsImV4cCI6MTcwNjUyNzY0NH0.AeQk23OPvlvauDEf45IlxxJ8ViSM5BlC6OlNkhXTomw", response.Token)
 }
 
+func TestLoginUserHandlerv2_InvalidHttpRequestMethod(t *testing.T) {
+	requestBody := []byte(`{
+		"username": 	"test_user",
+		"nonce": 		"Test_Nonce",
+		"c_nonce": 		"Test_Nonce",
+		"client_proof": "Test_Client_Proof"
+		}`)
+
+	req, err := http.NewRequest("GET", "/api/v2/login-user", bytes.NewBuffer(requestBody))
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	mockService := &MockService{
+		loginUserv2: func(req dto.LoginUserDTOv2) (models.LoginUserResponseOutputv2, error) {
+			return models.LoginUserResponseOutputv2{
+				Token:           "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VybmFtZSI6ImhtayIsInVzZXJfaWQiOjIsImlzcyI6Ikdsb2JlQW5kQ2l0aXplbiIsImV4cCI6MTcwNjUyNzY0NH0.AeQk23OPvlvauDEf45IlxxJ8ViSM5BlC6OlNkhXTomw",
+				ServerSignature: "Test_Server_Signature",
+			}, nil
+		},
+	}
```

Comment:
Similar to `LoginPrecheckHandler` tests, we can remove the underlying implementation from the test to make it look less complicated (this part is not expected to be executed by design in this case)